docs/LlpcOverview.md

```diff
@@ -0,0 +1,71 @@
+# LLPC overview
+
+This document outlines the state of LLPC as it will be at the end of the
+"middle-end interface clean-up" series of patches.
+
+## Compilation process
+
+The LLPC compilation process consists of three phases:
+* front-end;
+* middle-end (LGC);
+* back-end (LLVM AMDGPU back-end).
+
+### Front-end
```

Comment:
I don't why this tag isn't shown correctly in my local machine, could you double check it. thanks!

Reply:
As above; looks ok to me when browsing in github.